amplify/backend/amplify-meta.json

```diff
@@ -70,7 +70,11 @@
         "RootUrl": "https://n13ymupj66.execute-api.us-west-1.amazonaws.com/staging",
         "ApiId": "n13ymupj66"
       },
+<<<<<<< HEAD
+      "lastPushTimeStamp": "2022-03-11T18:44:12.800Z",
+=======
       "lastPushTimeStamp": "2022-03-11T18:33:52.116Z",
+>>>>>>> f077424bb6b82119491314bc2a311cf707f45615
```

Comment:
Looks like there are still 5 merge conflicts in this file

Reply:
lol oops